tests/test_pkg.py

```diff
@@ -0,0 +1,14 @@
+import unittest
+import modelcat.modelcatconnector as modelcatconnector
```

Comment:
I would suggest moving it to simply `connector` module, thus:
```suggestion
import modelcat.connector as modelcatconnector
```

Reply:
Resolved in https://github.com/Eta-Compute/modelcat/pull/1/commits/e6fd2378ee60404264cf60aa99251a3c3d9d97f7